hardhat.config.ts

```diff
@@ -162,12 +168,12 @@ const config: HardhatUserConfig = {
       kovan: process.env.ETHERSCAN_API_KEY!,
       rinkeby: process.env.ETHERSCAN_API_KEY!,
       goerli: process.env.ETHERSCAN_API_KEY!,
+      // sepolia: process.env.ETHERSCAN_API_KEY!, hardhat-etherscan is unhappy with custom definitions.
       optimisticEthereum: process.env.OPTIMISM_ETHERSCAN_API_KEY!,
       arbitrumOne: process.env.ARBITRUM_ETHERSCAN_API_KEY!,
       polygon: process.env.POLYGON_ETHERSCAN_API_KEY!,
       polygonMumbai: process.env.POLYGON_ETHERSCAN_API_KEY!,
     },
-    customChains: [],
```

Comment:
I had to remove the definition for `sepolia` because we inherit a very old version of `hardhat-etherscan` from `@uma/common`, and it doesn't support `customChains`.

I can't see any obvious way of preventing UMA's hardhat dependencies from executing, and intuitively it seems wrong that we inherit this requirement to _our_ environment from an external package. I've commented `sepolia` out for now, but it'd be good to address the underlying issue.

```
yarn run v1.22.19                                                                                               
$ hardhat compile && tsc && rsync -a --include '*/' --include '*.d.ts' --exclude '*' ./typechain ./dist/        
Error in plugin @nomiclabs/hardhat-etherscan: Etherscan API token "sepolia" is for an unsupported network                                                                                                                        
                                                                                                                
Learn more at https://hardhat.org/plugins/nomiclabs-hardhat-etherscan.html#multiple-api-keys-and-alternative-block-explorers                                                                                                     
                                                                                                                                                                                                                                 
NomicLabsHardhatPluginError: Etherscan API token "sepolia" is for an unsupported network                                                                                                                                         
                                                                                                                                                                                                                                 
Learn more at https://hardhat.org/plugins/nomiclabs-hardhat-etherscan.html#multiple-api-keys-and-alternative-block-explorers
```

Reply:
We actually should not being using `hardhat-etherscan` anymore to verify. The latest package we should be using is `hardhat-verify`: https://github.com/across-protocol/contracts-v2/blob/master/package.json#L53

Reply:
Correct - we're _not_ actually intending to use `hardhat-etherscan`. We seem to be inheiriting it via `@uma/common`, and I haven't found any way to inhibit that.